### DISCUSSION TITLE
Monaco editor UI add javascript type support

### DISCUSSION THREAD

**Discussion by ZuperZee** [grafana/grafana#38430] [Plugins]
Currently the monaco editor integrated into Grafana has removed typescript types.<https://github.com/grafana/grafana/blob/bafe16368ae945f4627e445280a63f53e2cfcf7a/scripts/webpack/webpack.common.js#L85-L88> Which makes it hard to use types.
![image](https://user-images.githubusercontent.com/39485579/130408305-ed349ead-2cca-4f7e-835f-0afd06a64f2b.png)

**Comment by ZuperZee** [comment]
Fixed in this PR #38703

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
